finesse/gui/temp_control.py

```diff
@@ -0,0 +1,359 @@
+            layout.addWidget(wid, 0, i)
+
+        self._ch1 = QLineEdit()
+        self._ch1.setReadOnly(True)
+        layout.addWidget(self._ch1, 1, 1)
+
+        self._ch2 = QLineEdit()
+        self._ch2.setReadOnly(True)
+        layout.addWidget(self._ch2, 1, 2)
+
+        self._ch3 = QLineEdit()
+        self._ch3.setReadOnly(True)
+        layout.addWidget(self._ch3, 1, 3)
+
+        self._ch4 = QLineEdit()
+        self._ch4.setReadOnly(True)
+        layout.addWidget(self._ch4, 1, 4)
+
+        self._ch5 = QLineEdit()
+        self._ch5.setReadOnly(True)
+        layout.addWidget(self._ch5, 1, 5)
+
+        self._ch6 = QLineEdit()
+        self._ch6.setReadOnly(True)
+        layout.addWidget(self._ch6, 1, 6)
+
+        self._ch7 = QLineEdit()
+        self._ch7.setReadOnly(True)
+        layout.addWidget(self._ch7, 1, 7)
+
+        self._ch8 = QLineEdit()
+        self._ch8.setReadOnly(True)
+        layout.addWidget(self._ch8, 1, 8)
+
+        wid = QLabel("POLL")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 0, 9, 2, 1)
+
+        self._poll_light = QLabel()
+        self._poll_light.setPixmap(QPixmap("./finesse/gui/images/poll_off.png"))
+        layout.addWidget(self._poll_light, 0, 10, 2, 1)
+
+        return layout
+
+
+class TC4820_HOT(QGroupBox):
+    """Widgets to view the TC4820 HOT properties."""
+
+    def __init__(self) -> None:
+        """Creates the widgets to control and monitor TC4820 HOT.
+
+        Returns:
+            None
+        """
+        super().__init__("TC4820 HOT")
+
+        layout = self._create_controls()
+        self.setLayout(layout)
+
+    def _create_controls(self) -> QGridLayout:
+
+        layout = QGridLayout()
+
+        wid = QLabel("CONTROL")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 0, 0)
+
+        wid = QLabel("POWER")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 1, 0)
+
+        wid = QLabel("SET")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 2, 0)
+
+        wid = QLabel("Pt 100")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 0, 2)
+
+        wid = QLabel("POLL")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 0, 4)
+
+        wid = QLabel("ALARM")
+        wid.setAlignment(Qt.AlignmentFlag.AlignRight | Qt.AlignmentFlag.AlignVCenter)
+        layout.addWidget(wid, 2, 4)
+
+        self._control_val = QLineEdit("70.5")
+        self._control_val.setReadOnly(True)
+        self._control_val.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        layout.addWidget(self._control_val, 0, 1)
+
+        self._pt100_val = QLineEdit("70.34")  # CH_7?
```

Comment:
Ah, this was just because I noticed that the value here corresponded to the value in the CH_7 box, so wondered whether the two components need to be connected.